src/appmenuwidget.cpp

```diff
@@ -735,9 +735,9 @@ void AppMenuWidget::searchEditingDone()
 {
     if (m_searchMenu && m_searchMenu->actions().count() > 1) {
         searchLineEdit->clearFocus();
-        for (QAction *findActivateeCanidcate : m_searchMenu->actions())
-            if (!findActivateeCanidcate->isSeparator()) {
-                m_searchMenu->setActiveAction(findActivateeCanidcate);
+        for (QAction *findCandidateAction : m_searchMenu->actions())
+            if (!findCandidateAction->isSeparator()) {
+                m_searchMenu->setActiveAction(findCandidateAction);
```

Comment:
this was a manual change of sorts. There were two tokens in this variable that weren't in the dictionary, it seemed to make sense to avoid the word `Active` as `findActive` could be confusing -- given the short scope, I opted to drop the token entirely in favor of `action` (I contemplated `menuitem` but I wasn't certain it was always going to be a menuitem, so I went with the type).